actions/asana-create-pr-subtask/asana-create-pr-subtask.sh

```diff
@@ -64,6 +64,24 @@ _check_pr_subtask_exist() {
 
 }
 
+# Gets the next business day (skips weekends)
+_get_next_business_day() {
+    local next_date
+    next_date=$(date -d "+1 day" "+%Y-%m-%d")
+    
+    # Check if the next day is a weekend
+    local day_of_week
+    day_of_week=$(date -d "$next_date" "+%u")
+    
+    if [ "$day_of_week" = "6" ]; then  # Saturday
+        next_date=$(date -d "$next_date +2 days" "+%Y-%m-%d")
+    elif [ "$day_of_week" = "7" ]; then  # Sunday
+        next_date=$(date -d "$next_date +1 day" "+%Y-%m-%d")
+    fi
```

Comment:
Nice 🙌

Reply:
BTW @samsymons  I didn’t look into automated testing of this as I believe this is low-risk to be tested by opening a test PR on the apple repo once this is merged and released.

Reply:
@aataraxiaa Yep, agreed with that approach, I think we're fine to test that way.